src/DIRAC/Core/Utilities/ClassAd/ClassAdLight.py

```diff
@@ -3,31 +3,30 @@
 """
```

Comment:
There's actually no need for this module to be in _src/DIRAC/Core/Utilities/ClassAd_, you can remove this directory and move the module in _src/DIRAC/Core/Utilities_